src/classic-handler.ts

```diff
@@ -1,4 +1,4 @@
-import * as Earthstar from "../../stone-soup/mod.ts";
+import * as Earthstar from "../../earthstar/mod.ts";
```

Comment:
Thanks for this! At this point I hadn't exposed the module as a URL yet. Let's change this to `https://deno.land/x/earthstar@v7.1.0/mod.ts`